libs/generic-view/store/src/lib/imports/contacts-mappers/outlook-api/map-outlook-api.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { UnifiedContact } from "device/models"
+import { Contact } from "Core/contacts/reducers/contacts.interface"
+import { getDisplayName } from "../helpers"
+
+export const mapOutlookApi = (contacts: Contact[]): UnifiedContact[] => {
+  try {
```

Comment:
I see that error in any of the contact entries replacing all contacts with an empty array. Maybe we can go in a different direction eg. corrupted contacts are skipped, so in a case when one out of 100 contacts is corrupted we still can process 99 correct contacts. TBD during standup meeting.